pkgs/by-name/nu/numbat/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "numbat";
+  version = "1.6.3";
+
+  src = fetchFromGitHub {
+    owner = "sharkdp";
+    repo = "numbat";
+    rev = "v${version}";
+    hash = "sha256-r6uPe2NL+6r/fKjf0C/5DLdB5YP3SIo8g8EsDxKP/3g=";
+  };
+
+  cargoHash = "sha256-MPqJjCfIwgK8QigWQYfWAYlg9RNMzF4x+0SprS0raKY=";
+
+  buildNoDefaultFeatures = true;
+
+  meta = with lib; {
+    description = "High precision scientific calculator with full support for physical units";
+    longDescription = ''
+      A statically typed programming language for scientific computations
+      with first class support for physical dimensions and units
+    '';
+    homepage = "https://numbat.dev";
+    changelog = "https://https://github.com/sharkdp/numbat/releases/tag/v${version}";
+    license = with licenses; [ asl20 /* or */ mit ];
+    mainProgram = "numbat";
+    maintainers = with maintainers; [ giomf ];
+    platforms = platforms.linux;
```

Comment:
I belive the convention is that `platform` should be set to what upstream specifies, or what we think is possible if nothing is specified. In this case, this program can probably run on a lot more that just `linux`. `rustPlatform` sets a default set of architectures which matches what rust can compile to, so I think this field can just be dropped in this case.

```suggestion
```